How-to-access-Oppia-webpages.md

```diff
@@ -1,237 +1,336 @@
-# How to access Oppia webpages
+## Table of Contents
+
+* [Log in](#log-in)
+* [Log in as a super-administrator](#log-in-as-a-super-administrator)
+  * [Using an email with implicit super-admin privileges](#using-an-email-with-implicit-super-admin-privileges)
+  * [Grant super-admin privileges after account creation](#grant-super-admin-privileges-after-account-creation)
+* [Access pages](#access-pages)
+  * [Admin page](#admin-page)
+    * [Assign roles](#assign-roles)
+  * [About page](#about-page)
+  * [Collection editor page](#collection-editor-page)
+  * [Collection player page](#collection-player-page)
+  * [Contact Page](#contact-page)
+  * [Creator dashboard page](#creator-dashboard-page)
+  * [Contributor dashboard page](#contributor-dashboard-page)
+  * [Delete account page](#delete-account-page)
+  * [Donate page](#donate-page)
+  * [Email dashboard page](#email-dashboard-page)
+  * [Exploration editor page](#exploration-editor-page)
+  * [Exploration player page](#exploration-player-page)
+  * [Get started page](#get-started-page)
+  * [Landing pages](#landing-pages)
+  * [Skill editor page](#skill-editor-page)
+  * [Story editor page](#story-editor-page)
+  * [Story player page](#story-player-page)
+  * [Topic editor (including preview tab)](#topic-editor-including-preview-tab)
+  * [Preferences page](#preferences-page)
+  * [Profile page](#profile-page)
+  * [Community library page](#community-library-page)
+  * [Topics and skills dashboard page](#topics-and-skills-dashboard-page)
+  * [Thanks page](#thanks-page)
+  * [Terms page](#terms-page)
```

Comment:
Should we divide these into the following sections:
1. Admin pages
    - Super admin page
    - Contributor dashboard admin page
    - Release coordinator page 
2. User account pages
    - Creating a user account
    - Creating admin account
    - Delete account page
    - Preferences page
    - Profile page
3. Editor pages
    - ...
4. Player pages
5. Contributor pages
6. Static pages

Reply:
done (though I used a slightly different grouping)